packages/components/forma-36-react-timepicker/package.json

```diff
@@ -16,19 +16,14 @@
     "lint": "tsdx lint"
   },
   "peerDependencies": {
-    "react": ">=16"
+    "react": "^16.8.6",
```

Comment:
Do we have to pin the peer version here? isn't that what is causing the duplicate react versions?

Reply:
I thought the duplication was caused by having it in devDependencies and here
but this was happening only on the website, no?

Reply:
yes, but `peerDependencies` is different, it's used only for consumer packages, it tells the consumer package i.e. "user_interface" which version of react it needs. it's not used for compilation

Reply:
So you are suggesting to keep devDep and removing peerDependencies totally, is that it?
Don't this mean that someone consuming react-timepicker that has no react will get an error?

Reply:
`peerDep` should stay, but with `>=16` just without pinning the version to avoid duplicate installation if consumer has `16.9.0` for example.
As for `devDep` i think we should keep it removed yes, should be able to get the react version from root I think.

Reply:
Ahhh I got it! I will make the change

Reply:
Thanks a lot 🖤

Reply:
It should still be `>=16.8` because of hooks.